projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/sharing-modal/index.tsx

```diff
@@ -253,7 +253,7 @@ const SharingModalInner: React.FC = () => {
 						<InlineSocialLogo icon="pinterest-alt" size={ 18 } />
 					</Button>
 					<div className="wpcom-block-editor-post-published-sharing-modal__checkbox-section">
-						<FormLabel htmlFor="toggle" className="is-checkbox">
+						<FormLabel htmlFor="toggle" className="is-checkbox" style={ { display: 'flex' } }>
```

Comment:
I think it would be preferable adding the CSS in the relevant file https://github.com/Automattic/jetpack/blob/94fb8a495afd16aa6947747736e91079097eba6f/projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/sharing-modal/style.scss#L116